src/component/Registration/SignIn.jsx

```diff
@@ -1,23 +1,42 @@
 import React from 'react';
-import { Link } from 'react-router-dom'; 
+import { Link } from 'react-router-dom';
 import './Signin.css';
 import NavBar from '../NavBar/NavBar'
 import Footer from '../Footer/Footer'
 import Input from './Input';
 
 
 const Signin = () => {
-  return <>
-           <NavBar />
-           <section className='registrationWrapper'>
-                <div className='registrationBody'>
-                    <form action="" className="registrationForm">
-                        <Input/>
-                    </form>
-                </div>
-            </section>
-            <Footer />
-  </>;
+    return <>
+        <NavBar />
+        <section className='signinWrapper'>
+            <div className='signinContainer'>
+                <form action="" className="signinForm">
+                    <div className='signinFormgroup'>
+                        <label htmlFor="Name">Username</label>
+                        <input type="text" name='name' />
+                    </div>
+
+                    <div className='signinFormgroup'>
+                        <label htmlFor="Password">Password</label>
+                        <input type="Password" id='name' name='Password' />
```

Comment:
Is it not necessary for this to have a place holder? If yes, lets add it. Please add for other input field also.